src/interlocking_component/route_controller.py

```diff
@@ -95,15 +74,58 @@ def insert_train_priority_changed(self, train: Train):
         raise NotImplementedError()
 
 
-class RouteController(IRouteController):
+class RouteController:
     """This class coordinates the route of a train.
     It calls the router to find a route for a train.
     It makes sure, that the Interlocking sets fahrstrassen along those routes.
     """
 
-    interlocking: interlockinginterface = None
+    interlocking: Interlocking = None
     router: Router = None
 
+    def start_interlocking(self, file_name: str = "test_example.ppxml"):
+        """This method instantiates the interlocking and the infrastructure_provider
+        and must be called before the interlocking can be used.
+        """
+        self.router = Router()
+
+        # Import from local PlanPro file
+        topology = PlanProReader(
+            "data/planpro/" + file_name
```

Comment:
I recommend you to use `path.join` for this 😇  It differentiates between OS if you have to use `/` or `\`.